### PR TITLE
misc cleanup of transformers #145

### LINKED ISSUE #145: Misc. cleanup of transformers
There are a few inconsistencies within the base transformer and state transformer on things like parameter names (`input_dict` vs `input_columns`) as well as some things we've fixed like `extract_precinct_split` not being in the base class but getting implemented in states.

After the merge into master, it'd be good to go through and resolve some of those inconsistencies in naming as well as documentation throughout. This would be a good beginner-friendly one to get familiar with the codebase.

- [x] input_dict
- [x] extract_precinct_split

### PR DESCRIPTION
**Pull request by slavingia**
# What's in this PR?

- input_columns -> input_dict
- made default of None for precinct split, it's often blank
- updated CO to use col_map so I could test

## References

Fixes #145

### CODE REVIEW

**Comment by pjsier** [comment]
Looks good, glad we're doing the cleanup and that `col_map` is getting used

File: src/python/national_voter_file/us_states/oh/transformer.py [hunk 1]
```diff
@@ -195,7 +195,7 @@ def extract_school_board_dist(self, input_dict):
 
     def extract_precinct_split(self, input_dict):
         # No split, copying precinct
```
Comment (pjsier) [inline src/python/national_voter_file/us_states/oh/transformer.py]: Comment can be removed since it's no longer applicable

**Comment by pjsier** [comment]
This all looks good. Going along with this, should we get rid of the `map_extract_by_keys` method on `BaseTransformer` now that it's covered with additional functionality by `col_map`?

**Comment by slavingia** [comment]
100%. created an issue for that.

**Pull request merged**